app/code/community/Bolt/Boltpay/Model/FeatureSwitch.php

```diff
@@ -24,13 +24,23 @@
  */
 class Bolt_Boltpay_Model_FeatureSwitch extends Bolt_Boltpay_Model_Abstract
 {
+    /**
+     * Set flag to true when bolt plugin is updated and we need to update features
+     */
+    private $needUpdateFeatures = false;
+
     /**
      * This method gets feature switches from Bolt and updates the local DB with
      * the latest values. To be used in upgrade data and webhooks.
      */
     public function updateSwitchesFromBolt()
```

Comment:
Slightly unrelated, but can we name this updateFeatureSwitchesFromBolt?

Reply:
Done